docs/fidelity-bonds.md

```diff
@@ -206,6 +206,8 @@ miner fees, you can probably wait until fees are low).
 The full details on valuing a time-locked fidelity bond are [found in the relevant section of the
 "Financial mathematics of fidelity bonds" document](https://gist.github.com/chris-belcher/87ebbcbb639686057a389acb9ab3e25b#time-locked-fidelity-bonds).
 
+To see how valuable a bond would be, and to compare it with the orderbook, you can use the `bond-calculator.py` script.
```

Comment:
nit: probably mention the location of the script and/or give an example invocation

Reply:
Absolutely, I'll keep this for last, once the rest is more or less stable.

Reply:
This was forgotten?

Reply:
I'm waiting to see if this is actually how we want to proceed, in particular the orderbook thing (is the JSON file okay?)

Reply:
72bf4474ddb279b39a00506710e3d3acf50948ce adds a possible version.